puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -10,18 +10,23 @@ pipeline {
         ansiColor('xterm')
     }
 
-    environment {
-        ruby_version = "2.5"
-    }
-
     stages {
         stage('Copy Source') {
             steps {
                 script {
-                    sourcefile_paths = pwd(tmp: true)
+                    artifact_path = "${pwd()}/artifacts"
                     source_project_name = "${project_name}-${git_ref}-source-release"
-                    copyArtifacts(projectName: source_project_name, target: sourcefile_paths, selector: {lastSuccessful: true})
+                    copyArtifacts(projectName: source_project_name, target: artifact_path)
```

Comment:
There's no working selector here?

Reply:
In the end, I couldn't figure out what it needed to work properly and the default was what we want (last successful build of the target project).